src/auth/computeclient.ts

```diff
@@ -18,7 +18,9 @@ import {AxiosError, AxiosPromise, AxiosRequestConfig, AxiosResponse} from 'axios
 
 import {RequestError} from './../transporters';
 import {CredentialRequest, Credentials} from './credentials';
-import {GetTokenResponse, OAuth2Client} from './oauth2client';
+import {GetTokenResponse, OAuth2Client, RefreshOptions} from './oauth2client';
+
+export interface ComputeOptions extends RefreshOptions {}
```

Comment:
Do we need to create specific ComputeOptions here?  Why not just use RefreshOptions directly?

Reply:
Done.

Reply:
I see value in a separate type in case `ComputeOptions` adds more properties in the future. Having an empty type alias here avoids a potential breaking change in the future.

Reply:
@ofrobots, @JustinBeckwith -- What's the decision here?
I'd originally added the separate types for the reasons ofrobots suggested, but I have no strong opinions here.

Reply:
I am happy to be outvoted here :)  I don't feel terribly strongly either.